ts/session/apis/open_group_api/opengroupV2/JoinOpenGroupV2.ts

```diff
@@ -154,25 +159,25 @@ export async function joinOpenGroupV2WithUIEvents(
 
     uiCallback?.({ loadingState: 'started', conversationKey: conversationID });
 
-    await joinOpenGroupV2(parsedRoom, fromConfigMessage);
+    const convoCreated = await joinOpenGroupV2(parsedRoom, fromConfigMessage);
 
-    const isConvoCreated = getConversationController().get(conversationID);
+    const isConvoCreated = Boolean(convoCreated);
```

Comment:
Do we need the variable assignment here? Can we not just put it in the if statement?

Reply:
fixed

Reply:
👍